gdk/wizard/commands/data.py

```diff
@@ -0,0 +1,53 @@
+from gdk.common.configuration import get_configuration, _get_project_config_file
+import gdk.common.consts as consts
+import gdk.common.utils as utils
+import json
+
+
+class Wizard_data:
+    def __init__(self):
+        self.project_config_file = "/Users/jacksozh/Desktop/aws-greengrass-gdk-cli/gdk/wizard/static/default_config.json"
```

Comment:
What if file exists? Also, you are specifying your local path which will not be present on customer's system.